pkg/register/component.go

```diff
@@ -5,16 +5,20 @@ import (
 	"errors"
 	"time"
 
-	"github.com/cloudtrust/keycloak-client/v2/toolbox"
-
 	"github.com/cloudtrust/common-service/v2/configuration"
 	"github.com/cloudtrust/common-service/v2/database"
 	errorhandler "github.com/cloudtrust/common-service/v2/errors"
 	"github.com/cloudtrust/common-service/v2/validation"
+	apicommon "github.com/cloudtrust/keycloak-bridge/api/common"
 	apiregister "github.com/cloudtrust/keycloak-bridge/api/register"
 	"github.com/cloudtrust/keycloak-bridge/internal/constants"
 	"github.com/cloudtrust/keycloak-bridge/internal/keycloakb"
 	kc "github.com/cloudtrust/keycloak-client/v2"
+	"github.com/cloudtrust/keycloak-client/v2/toolbox"
+)
+
+const (
+	frontendType = "register"
```

Comment:
Here again, I think this should be put in the endpoint

Reply:
Already answered for account/component.go